docs/tools/toolchains/export_to_third_party.md

```diff
@@ -97,17 +94,17 @@ make: *** [all] Error 2
 
 If you see such an error, remove the `-Wl,--wrap,_malloc_r`, `-Wl,--wrap,_free_r`, `-Wl,--wrap,_realloc_r` and `-Wl,--wrap,_calloc_r`from the `LD_FLAGS` variable in the Makefile.
 
-### Make and Eclipse (GNU Arm Embedded Toolchain, IAR Compiler, Arm Compiler 5)
+### Make and Eclipse (GNU Arm Embedded Toolchain, IAR Compiler, Arm Compiler 6)
 
 <span class="notes">**Note:** Our Eclipse CDT projects use Makefile. Therefore, Makefile advice also applies to using Eclipse.</span>
 
 Make itself does not compile source code. It relies on a compiler such as:
 
 - [GNU Arm Embedded Toolchain](https://developer.arm.com/open-source/gnu-toolchain/gnu-rm), which you can install for free using the [instructions](https://gnu-mcu-eclipse.github.io/toolchain/arm/install/). Please note that the current Makefile requires that you add your compiler to your PATH variable. This contradicts the instruction given on the installation website, because those instructions are intended for Eclipse, not Make.
 - The IAR Arm compiler bundled with the IAR Embedded Workbench.
-- Arm Compiler 5.
+- Arm Compiler 6.
 
-<span class="notes">**Note:** Ensure that the compiler you are exporting to is accessible using your `PATH` environment variable because Makefile requires this. For example, when using an exported Makefile from `make_armc5`, the command `armcc` prints a help message about how to use Arm Compiler 5.</span>
+<span class="notes">**Note:** Ensure that the compiler you are exporting to is accessible using your `PATH` environment variable because Makefile requires this. For example, when using an exported Makefile from `make_armc6`, the command `armcc` prints a help message about how to use Arm Compiler 6.</span>
```

Comment:
There is no `armcc` with Arm Compiler 6. It has been replaced with `armclang`.

Reply:
```suggestion
<span class="notes">**Note:** Ensure that the compiler you are exporting to is accessible using your `PATH` environment variable because Makefile requires this. For example, when using an exported Makefile from `make_armc6`, the command `armclang` prints a help message about how to use Arm Compiler 6.</span>
```